packages/vhd-lib/src/Vhd/VhdDirectory.js

```diff
@@ -172,8 +175,8 @@ export class VhdDirectory extends VhdAbstract {
 
   async coalesceBlock(child, blockId) {
     assert.strictEqual(this._handler, child._handler)
-    this._handler.copy(child._getBlockPath(blockId), this.getChunkPath(blockId))
-    return await this._handler.getSize(this.getChunkPath(blockId))
+    this._handler.copy(child.getChunkPath(child._getBlockPath(blockId)), this.getChunkPath(this._getBlockPath(blockId)))
```

Comment:
Please respond to this comment :slightly_smiling_face: 

It can be fixed in another PR, simply in a separate commit, or we may simply decide to keep it public, but I don't want this comment to be forgotten.

Reply:
it have to be called from another vhd , so I don't understand how it can be private (like a real private method). Are you thinking  of  'private by convention' with a _ ?

Reply:
IIRC private fields are private by class, not by instance :slightly_smiling_face: 

And if I'm mistaken, yes let's use a `_` prefix, I want to make it clear that it's not part of the public API.